core/commands/checkout.py

```diff
@@ -126,10 +126,19 @@ class GsCheckoutCurrentFileAtCommitCommand(LogMixin, WindowCommand, GitCommand):
     Reset the current active file to a given commit.
     """
 
+    show_commit_info = False
```

Comment:
What is this for?

Reply:
LOL? It's your code.

If you don't set `show_commit_info` to `False` it will run a default action on highlight, namely `gs_show_commit_info`.

https://github.com/divmain/GitSavvy/blob/92bc8bfcd3eaff93a17a7bb3480e106bfd802e17/core/ui_mixins/quick_panel.py#L514-L526

Reply:
haha, just checking you :D 
Apparently isn't obvious why so refactored it a little for you (and me in 3 months)
https://github.com/divmain/GitSavvy/pull/957

Reply:
Now it's not needed anymore. All you need to do is to overite on_highlight

Reply:
No, that doesn't work. Now `on_highlight` is never called unless I implement `run_async` and pass my `on_highlight` down. Apparently #957 is not a refactoring that makes the program flow any clearer.